rust/src/connection/network/transmitter/rpc.rs

```diff
@@ -26,52 +26,37 @@ use tokio::{
 };
 
 use super::{oneshot_blocking, response_sink::ResponseSink};
-use crate::{
-    common::{address::Address, RequestID, Result},
-    connection::{
-        message::{Request, Response},
-        network::{
-            channel::{open_callcred_channel, open_plaintext_channel, GRPCChannel},
-            proto::{FromProto, IntoProto, TryFromProto, TryIntoProto},
-            stub::RPCStub,
-        },
-        runtime::BackgroundRuntime,
+use crate::{common::{address::Address, RequestID, Result}, connection::{
+    message::{Request, Response},
+    network::{
+        channel::{open_callcred_channel, GRPCChannel},
+        proto::{FromProto, IntoProto, TryFromProto, TryIntoProto},
+        stub::RPCStub,
     },
-    Credential, Error,
-};
+    runtime::BackgroundRuntime,
+}, ConnectionSettings, Credential, Error};
 
 pub(in crate::connection) struct RPCTransmitter {
     request_sink: UnboundedSender<(Request, ResponseSink<Response>)>,
     shutdown_sink: UnboundedSender<()>,
 }
 
 impl RPCTransmitter {
-    pub(in crate::connection) fn start_core(address: Address, runtime: &BackgroundRuntime) -> Result<Self> {
-        let (request_sink, request_source) = unbounded_async();
-        let (shutdown_sink, shutdown_source) = unbounded_async();
-        runtime.run_blocking(async move {
-            let channel = open_plaintext_channel(address);
-            let rpc = RPCStub::new(channel, None).await;
-            tokio::spawn(Self::dispatcher_loop(rpc, request_source, shutdown_source));
-            Ok::<(), Error>(())
-        })?;
-        Ok(Self { request_sink, shutdown_sink })
-    }
-
-    pub(in crate::connection) fn start_cloud(
+    pub(in crate::connection) fn start(
         address: Address,
         credential: Credential,
+        connection_settings: ConnectionSettings,
         runtime: &BackgroundRuntime,
     ) -> Result<Self> {
-        let (request_sink, request_source) = unbounded_async();
-        let (shutdown_sink, shutdown_source) = unbounded_async();
+        let (req_sink, req_src) = unbounded_async();
```

Comment:
I know that we sometimes use shorter versions for names in our codestyle where it's clear what's the full name is (`req` -> `request`, `tx` -> `transaction`, etc.), but I also heard from @flyingsilverfin that we want to use full named if possible. Not sure if I'm a fan of this refactor here, kinda introduces codestyle inconsistencies!

Reply:
Reverted in 055197fc83d9b0bd2a457f39488c063a75ad6e05